internal/handlers/common/error.go

```diff
@@ -28,6 +28,8 @@ import (
 type ErrorCode int32
 
 const (
+	errUnset = ErrorCode(0) //
```

Comment:
```suggestion
	errUnset = ErrorCode(0) // Unset
```

And run `task gen` after that

Reply:
@fenogentov why didn't you commit that change? for stringer comments matter

Reply:
> @fenogentov why didn't you commit that change? for stringer comments matter

mongo has this error with no name
*fixed for similar behavior